lib/usecases/run/UpdateRunUseCase.js

```diff
@@ -126,6 +129,24 @@ class UpdateRunUseCase {
                 if (eorReasons) {
                     await this._updateEorReasonsOnRun(runId, eorReasons);
                 }
+                if (tags) {
+                    const existingTags = await new GetAllTagsUseCase()
+                        .execute({ query: { page: { filter: { ids: tags } } } });
+                    const missingTags = tags.reduce((accumulator, tag) => {
+                        if (!existingTags.tags.find(({ id }) => id === tag)) {
+                            accumulator.push(tag);
```

Comment:
The tests do not cover invalid tags yet.

Reply:
This is the first step of a refactoring, I add test in the next PR

Reply:
If the tests work on the next branch locally it is not a problem.

Reply:
Yes, hey pass locally for the next branch